packages/terra-application-navigation/tests/jest/extensions/ExtensionRollup.test.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import ExtensionRollup from '../../../src/extensions/_ExtensionRollup';
+import mockIntl from '../mockIntl';
```

Comment:
mockIntl is provided in the terra-enzyme-intl package: https://github.com/cerner/terra-enzyme-intl#mockintl

Reply:
Updated.